src/lib/libmints/molecule.cc

```diff
@@ -245,6 +234,18 @@ Molecule& Molecule::operator=(const Molecule& other)
         atoms_.push_back((*iter)->clone(atoms_, geometry_variables_));
     }
 
+
+    // These are symmetry related variables, and are filled in by the following functions
+    pg_             = boost::shared_ptr<PointGroup>(new PointGroup(other.pg_));
```

Comment:
Could it be something to do with the fact that the pg will change if we
ghost some atoms?

On Thu, Mar 3, 2016 at 8:07 AM, Daniel Smith <notifications@github.com>
wrote:

> In src/lib/libmints/molecule.cc
> <https://github.com/psi4/psi4/pull/299#discussion_r54876571>:
>
> > @@ -245,6 +234,18 @@ Molecule& Molecule::operator=(const Molecule& other)
> >          atoms_.push_back((*iter)->clone(atoms_, geometry_variables_));
> >      }
> >
> > +
> > +    // These are symmetry related variables, and are filled in by the following functions
> > +    pg_             = boost::shared_ptr<PointGroup>(new PointGroup(other.pg_));
>
> I recall there is a reason we do not copy the PG. @loriab
> <https://github.com/loriab> Do you remember what this is?
>
> —
> Reply to this email directly or view it on GitHub
> <https://github.com/psi4/psi4/pull/299/files#r54876571>.
>